view/readrom.js

```diff
@@ -76,37 +74,53 @@ const urlGetParam = new URLSearchParams(window.location.search);
 var advancedMode = urlGetParam.get("advanced") === "true";
 
 if (advancedMode) {
-    $("#romload").hide();
+    // $("#romload").hide();
```

Comment:
The purpose was to prevent having a request to the server if we were about to use local file